File: test/fixtures/kotlin/multiline_post_with_data.kt
import java.io.IOException
import okhttp3.FormBody
import okhttp3.OkHttpClient
import okhttp3.Request

val client = OkHttpClient()

val formBody = FormBody.Builder()
  .add("msg1", "value1")
  .add("msg2", "value2")
  .build()

val request = Request.Builder()
  .url("http://localhost:28139/echo/html/")
  .method("GET", formBody)
  .header("Origin", "http://fiddle.jshell.net")
  .header("Content-Type", "application/x-www-form-urlencoded")
  .build()

client.newCall(request).execute().use { response ->
  if (!response.isSuccessful) throw IOException("Unexpected code $response")
  response.body!!.string()
}

File: test/fixtures/perl/multiline_post_with_data.pl
use LWP::UserAgent;

$ua = LWP::UserAgent->new();
$response = $ua->get(
    'http://localhost:28139/echo/html/',
    Origin => 'http://fiddle.jshell.net',
    'Content-Type' => 'application/x-www-form-urlencoded',
    Content => 'msg1=value1&msg2=value2'
);
